internal/sorter/api.go

```diff
@@ -0,0 +1,135 @@
+package sorter
+
+import (
+	"backend/internal/entity"
+	"backend/pkg/db"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/gorilla/mux"
+)
+
+func ClassifyHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
would be nice to seperate some logic out to other functions

Reply:
🗿